editor/export/editor_export_preset.cpp

```diff
@@ -332,4 +333,29 @@ Variant EditorExportPreset::get_or_env(const StringName &p_name, const String &p
 	return get(p_name, r_valid);
 }
 
+String EditorExportPreset::get_version(const StringName &p_preset_string, bool p_windows_version) {
+	String result = get(p_preset_string);
+	if (result.is_empty()) {
+		result = GLOBAL_GET("application/config/version");
+
+		if (p_windows_version) {
+			// Modify version number to match Windows constraints (version numbers must have 4 components).
+			const PackedStringArray result_split = result.split(".");
+			String windows_version;
+			if (result_split.size() <= 1) {
+				result = result + ".0.0.0";
```

Comment:
This doesn't seem to handle empty strings, which would lead to `.0.0.0` version number on Windows.